src/globals/defines.php

```diff
@@ -61,7 +61,8 @@
 const BUILD_TARGET_MICRO = 2;   // build micro
 const BUILD_TARGET_FPM = 4;     // build fpm
 const BUILD_TARGET_EMBED = 8;   // build embed
-const BUILD_TARGET_ALL = 15;    // build all
+const BUILD_TARGET_DEV = 16;    // build dev
```

Comment:
should get rid of this, there's no point in an extra build target. embed sapi already builds everything that's needed and duplicating this serves no good purpose